transports/bifrost-http/handlers/config.go

```diff
@@ -124,8 +134,87 @@ func (h *ConfigHandler) updateConfig(ctx *fasthttp.RequestCtx) {
 	}
 
 	ctx.SetStatusCode(fasthttp.StatusOK)
-	SendJSON(ctx, map[string]any{
-		"status":  "success",
-		"message": "configuration updated successfully",
-	}, h.logger)
+	SendJSON(ctx, map[string]string{"status": "success"}, h.logger)
+}
+
+// getVectorStoreConfig handles GET /api/config/vector-store - Get the current vector store configuration
+func (h *ConfigHandler) getVectorStoreConfig(ctx *fasthttp.RequestCtx) {
```

Comment:
lets use GetVectorStoreConfigRedacted from config instead of sending down raw config + the function GetVectorStoreConfigRedacted also does not have a case for VectorStoreTypeRedis so will need to add that as well